mods/gennext/README.md

```diff
@@ -210,31 +210,53 @@ New mechanic: Intrinsics:
 - Pokémon that previously get Levitate are now immune to Ground intrinsically.
   Instead, they get new abilities in addition to their Ground immunity:
 
-  - Unown: Shadow Tag
+  - Azelf: Steadfast
 
-  - Bronzong: Heatproof
+  - Bronzong: None (its other two abilities remain available)
 
-  - Flygon: Compoundeyes, Sand Rush
+  - Carnivine: Strong Jaw, Dry Skin
+
+  - Chimecho: Soundproof, Magic Bounce
+
+  - Claydol: Filter, Keen Eye
 
-  - Weezing: Aftermath
+  - Cresselia: Natural Cure
+
+  - Cryogonal: Ice Body, Snow Cloak
+
+  - Eelektross: Poison Heal, Static
+
+  - Flygon: Compoundeyes, Sand Rush
 
-  - Eelektross: Poison Heal
+  - Gengar: Cursed Body, Unnerve
 
-  - Claydol: Filter
+  - Hydreigon: Sheer Force, Unnerve
 
-  - Cryogonal: Ice Body
+  - Latias: Pressure
 
-  - Gengar: Cursed Body
+  - Latios: Pressure
 
-  - Mismagius: Cursed Body
+  - Lunatone: Sturdy, Magic Guard
 
   - Mesprit: Serene Grace
 
-  - Uxie: Synchronize
+  - Mismagius: Cursed Body, Serene Grace
 
-  - Azelf: Steadfast
+  - Rotom: Motor Drive, plus a hidden ability based on forme:
+    - Rotom: Adaptability
+    - Rotom-Heat: Flash Fire
+    - Rotom-Wash: Storm Drain
+    - Rotom-Mow: Sap Sipper
+    - Rotom-Frost: Snow Warning
+    - Rotom-Spin: Cloud Nine
```

Comment:
"Spin" is the Japanese name, like "Cut" is for "Mow" and the other three formes happened to be localized with their Japanese names. This is actually a good reminder to add those two to aliases since people do still use them out of habit. I'll add them when I get back in several hours unless someone beats me to it.